color/p5js/Law of simmilarity + color/index.js

```diff
@@ -1,11 +1,15 @@
+canvas.parent('sketch');
```

Comment:
This has to be called after assigning the return value of `createCanvas` to the variable `canvas`